projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -42,6 +45,12 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   @Input('clrAccordionPanelDisabled') @HostBinding('class.clr-accordion-panel-disabled') disabled = false;
   @Input('clrAccordionPanelOpen') panelOpen = false;
 
+  @Input('clrAccordionPanelHeadingEnabled') headingEnabled = false;
+  /**
+   * Level of the accordion/stepper heading from 1 to 6.
```

Comment:
```suggestion
   * Level of the accordion/stepper heading from 1 to 6. This only works if `[clrAccordionPanelHeadingEnabled]` is set to `true`.
```

I just realized that this documentation comment is incomplete. We might also want to document `[clrAccordionPanelHeadingEnabled]` as well.